src/form/v2/birth/forms/printForm/index.ts

```diff
@@ -0,0 +1,247 @@
+  not,
+  PageTypes
+} from '@opencrvs/toolkit/events'
+import { applicationConfig } from '@countryconfig/api/application/application-config'
+import { printCertificateCollectors } from './collectors'
+import { printCertificateCollectorOther } from './collector-other'
+import { printCertificateCollectorIdentityVerify } from './collector-identity-verify'
+
+export const BIRTH_CERTIFICATE_COLLECTOR_FORM = defineActionForm({
+  label: {
+    id: 'v2.event.birth.action.certificate.form.label',
+    defaultMessage: 'Birth certificate collector',
+    description: 'This is what this form is referred as in the system'
+  },
+  pages: [
+    {
+      id: 'collector',
+      type: PageTypes.enum.FORM,
+      title: {
+        id: 'v2.event.birth.action.certificate.form.section.who.title',
+        defaultMessage: 'Certify record',
+        description: 'This is the title of the section'
+      },
+      fields: [...printCertificateCollectors, ...printCertificateCollectorOther]
+    },
+    {
+      id: 'collector.identity.verify',
+      type: PageTypes.enum.VERIFICATION,
+      title: {
+        id: 'event.birth.action.print.verifyIdentity',
+        defaultMessage: 'Verify their identity',
+        description: 'This is the title of the section'
+      },
+      fields: printCertificateCollectorIdentityVerify,
+      actions: {
+        verify: {
+          label: {
+            defaultMessage: 'Verified',
+            description: 'This is the label for the verification button',
+            id: 'v2.event.birth.action.certificate.form.verify'
+          }
+        },
+        cancel: {
+          label: {
+            defaultMessage: 'Identity does not match',
+            description:
+              'This is the label for the verification cancellation button',
+            id: 'v2.event.birth.action.certificate.form.cancel'
+          },
+          confirmation: {
+            title: {
+              defaultMessage: 'Print without proof of ID?',
+              description:
+                'This is the title for the verification cancellation modal',
+              id: 'v2.event.birth.action.certificate.form.cancel.confirmation.title'
+            },
+            body: {
+              defaultMessage:
+                'Please be aware that if you proceed, you will be responsible for issuing a certificate without the necessary proof of ID from the collector',
+              description:
+                'This is the body for the verification cancellation modal',
+              id: 'v2.event.birth.action.certificate.form.cancel.confirmation.body'
+            }
+          }
+        }
+      }
+    },
+    {
+      id: 'collector.collect.payment',
+      type: PageTypes.enum.FORM,
+      title: {
+        id: 'event.birth.action.print.collectPayment',
+        defaultMessage: 'Collect Payment',
+        description: 'This is the title of the section'
+      },
+      fields: [
+        {
+          id: 'collector.collect.payment.data.after365',
```

Comment:
This could be renamed `collector.collect.payment.data.afterLateRegistration` or such, since the late reg configuration could be something else than 365 days

Sorry my comment in the previous PR was off 😅